### ISSUE TITLE
Building 2.092.0 fails

### ISSUE THREAD

**Issue by Robert-M-Muench** [CyberShadow/Digger#84]
I built the latest Digger version as 64 bit and tried to build DMD 2.092.0 then. This is the error I get:

```
"C:\tmp\digger-win\work\dl\vs2013-community\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\x86_amd64\lib.exe" /nologo /OUT:zlib64.lib adler32.obj compress.obj crc32.obj deflate.obj gzclose.obj gzlib.obj gzread.obj  gzwrite.obj infback.obj inffast.obj inflate.obj inftrees.obj trees.obj uncompr.obj zutil.obj


cd ..\..\..
"C:\tmp\digger-win\work\build\bin\dmd.exe" -lib -ofphobos64.lib -Xfphobos.json -conf= -m64 -O -release -w -de -preview=dip1000 -transition=complex -I../druntime\import std\stdio.d  std\string.d  std\format.d  std\file.d  std\array.d  std\functional.d  std\path.d  std\outbuffer.d  std\utf.d  std\csv.d  std\complex.d  std\numeric.d  std\bigint.d  std\math.d  std\uni.d  std\base64.d  std\ascii.d  std\demangle.d  std\uri.d  std\mmfile.d  std\getopt.d  std\signals.d  std\meta.d  std\typetuple.d  std\traits.d  std\encoding.d  std\xml.d  std\random.d  std\exception.d  std\compiler.d  std\system.d  std\concurrency.d  std\bitmanip.d  std\typecons.d  std\uuid.d  std\variant.d  std\zlib.d  std\socket.d  std\conv.d  std\zip.d  std\stdint.d  std\json.d  std\parallelism.d  std\mathspecial.d  std\process.d  std\package.d  std\algorithm\package.d  std\algorithm\comparison.d  std\algorithm\iteration.d  std\algorithm\mutation.d  std\algorithm\searching.d  std\algorithm\setops.d  std\algorithm\sorting.d  std\algorithm\internal.d  std\container\array.d  std\container\binaryheap.d  std\container\dlist.d  std\container\rbtree.d  std\container\slist.d  std\container\util.d  std\container\package.d  std\datetime\date.d  std\datetime\interval.d  std\datetime\package.d  std\datetime\stopwatch.d  std\datetime\systime.d  std\datetime\timezone.d  std\digest\crc.d  std\digest\sha.d  std\digest\md.d  std\digest\ripemd.d  std\digest\digest.d  std\digest\hmac.d  std\digest\murmurhash.d  std\digest\package.d  std\net\isemail.d  std\net\curl.d  std\range\package.d  std\range\primitives.d  std\range\interfaces.d  std\regex\internal\ir.d  std\regex\package.d  std\regex\internal\parser.d  std\regex\internal\tests.d  std\regex\internal\tests2.d  std\regex\internal\backtracking.d  std\regex\internal\thompson.d  std\regex\internal\kickstart.d  std\regex\internal\generator.d    std\windows\registry.d  std\windows\syserror.d  std\windows\charset.d    std\internal\cstring.d  std\internal\unicode_tables.d  std\internal\unicode_comp.d  std\internal\unicode_decomp.d  std\internal\unicode_grapheme.d  std\internal\unicode_norm.d  std\internal\scopebuffer.d  std\internal\test\dummyrange.d  std\internal\test\range.d  std\internal\digest\sha_SSSE3.d  std\internal\math\biguintcore.d  std\internal\math\biguintnoasm.d  std\internal\math\biguintx86.d  std\internal\math\gammafunction.d  std\internal\math\errorfunction.d  std\internal\windows\advapi32.d  std\experimental\checkedint.d std\experimental\typecons.d  std\experimental\allocator\common.d  std\experimental\allocator\gc_allocator.d  std\experimental\allocator\mallocator.d  std\experimental\allocator\mmap_allocator.d  std\experimental\allocator\showcase.d  std\experimental\allocator\typed.d  std\experimental\allocator\package.d  std\experimental\allocator\building_blocks\affix_allocator.d  std\experimental\allocator\building_blocks\aligned_block_list.d  std\experimental\allocator\building_blocks\allocator_list.d  std\experimental\allocator\building_blocks\ascending_page_allocator.d  std\experimental\allocator\building_blocks\bitmapped_block.d  std\experimental\allocator\building_blocks\bucketizer.d  std\experimental\allocator\building_blocks\fallback_allocator.d  std\experimental\allocator\building_blocks\free_list.d  std\experimental\allocator\building_blocks\free_tree.d  std\experimental\allocator\building_blocks\kernighan_ritchie.d  std\experimental\allocator\building_blocks\null_allocator.d  std\experimental\allocator\building_blocks\quantizer.d  std\experimental\allocator\building_blocks\region.d  std\experimental\allocator\building_blocks\scoped_allocator.d  std\experimental\allocator\building_blocks\segregator.d  std\experimental\allocator\building_blocks\stats_collector.d  std\experimental\allocator\building_blocks\package.d  std\experimental\logger\core.d  std\experimental\logger\filelogger.d  std\experimental\logger\multilogger.d  std\experimental\logger\nulllogger.d  std\experimental\logger\package.d    etc\c\zlib.d  etc\c\curl.d  etc\c\sqlite3.d  etc\c\odbc\sql.d  etc\c\odbc\sqlext.d  etc\c\odbc\sqltypes.d  etc\c\odbc\sqlucode.d  etc\c\zlib\zlib64.lib ../druntime/lib/druntime64.lib
std\stdio.d(16): Error: module stddef is in file 'core\stdc\stddef.d' which cannot be read
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
std\format.d(60): Error: module vararg is in file 'core\vararg.d' which cannot be read
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
std\stdio.d(126): Error: module basetsd is in file 'core\sys\windows\basetsd.d' which cannot be read
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
std\stdio.d(16): Error: module stddef is in file 'core\stdc\stddef.d' which cannot be read
import path[0] = ../druntime\import
std\stdio.d(17): Error: module stdio is in file 'core\stdc\stdio.d' which cannot be read
import path[0] = ../druntime\import
std\stdio.d(126): Error: module basetsd is in file 'core\sys\windows\basetsd.d' which cannot be read
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'
       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
       config file: not found
import path[0] = ../druntime\import
Error: cannot find source code for runtime library file 'object.d'

--- errorlevel 1
digger: Saving to cache.
digger: Clearing temporary cache

object.Exception@ae\sys\d\manager.d(858): Command ["make", "-f", "win64.mak", "phobos64.lib", "DMD=C:\\tmp\\digger-win\\work\\build\\bin\\dmd.exe", "MODEL=64", "VCDIR=C:\\tmp\\digger-win\\work\\dl\\vs2013-community\\Program Files (x86)\\Microsoft Visual Studio 12.0\\VC", "SDKDIR=C:\\tmp\\digger-win\\work\\dl\\vs2013-community\\Program Files\\Microsoft SDKs\\Windows\\v7.1A", "CC=C:\\tmp\\digger-win\\work\\dl\\vs2013-community\\Program Files (x86)\\Microsoft Visual Studio 12.0\\VC\\bin\\x86_amd64\\cl.exe", "LD=C:\\tmp\\digger-win\\work\\dl\\vs2013-community\\Program Files (x86)\\Microsoft Visual Studio 12.0\\VC\\bin\\x86_amd64\\link.exe", "AR=C:\\tmp\\digger-win\\work\\dl\\vs2013-community\\Program Files (x86)\\Microsoft Visual Studio 12.0\\VC\\bin\\x86_amd64\\lib.exe"] failed with status 1
----------------
0x00007FF7487CFE63
0x00007FF7487CFD8B
0x00007FF74889F11E
0x00007FF7488A5402
0x00007FF7488A4623
0x00007FF74889CA22
0x00007FF74889D28F
0x00007FF7488A9CBC
0x00007FF74893FB96
0x00007FF7489403DE
0x00007FF7487C1146
0x00007FF748870E1C
0x00007FF74886EB13
0x00007FF748885A8D
0x00007FF74886E932
0x00007FF7487C21CD
0x00007FF7487C282D
0x00007FF7487EC26F
0x00007FF7487C2864
0x00007FF748965873
0x00007FF7489656FC
0x00007FF7489657DB
0x00007FF7489656FC
0x00007FF748965643
0x00007FF74894A280
0x00007FF7487D6A24
0x00007FF7489CF2B0
0x00007FFE4C857BD4 in BaseThreadInitThunk
0x00007FFE4D5ECE51 in RtlUserThreadStart
C:\tmp\digger-win>
```

Any ideas what the problem could be?

**Comment by Robert-M-Muench** [comment]
./work/build/import/core only contains sync/ folder nothing else...

**Comment by CyberShadow** [comment]
Could you please try and see if it still happens with a clean work dir? (I.e. let it redownload and rebuild everything from zero)

**Comment by Robert-M-Muench** [comment]
Unfortunately, I get the same result.

**Comment by CyberShadow** [comment]
Sorry for the delay, I forgot that this is still an issue.

This was caused by https://github.com/dlang/druntime/pull/3026.

Fix added in Digger 3.0.5.
